File: Programmers/Lv.0/문자열 여러 번 뒤집기/181913.js
function solution(my_string, queries) {
    let answer = '';
    
    const str = [...my_string];
    for (const [s, e] of queries) {
        for (let i=0; i<=(e-s)/2; i++) {
            const t = str[s + i];
            str[s + i] = str[e - i];
            str[e - i] = t;
        }
    }
    
    answer = str.join('');
    
    return answer;
}

File: Programmers/Lv.0/문자열 여러 번 뒤집기/181913.java
class Solution {
    public String solution(String my_string, int[][] queries) {
        String answer = "";
        
        String[] str = my_string.split("");
        for (int[] query : queries) {
            int s = query[0];
            int e = query [1];
            for (int i=0; i<=(e-s)/2; i++) {
                String t = str[s + i];
                str[s + i] = str[e - i];
                str[e - i] = t;
            }
        }
        
        answer = String.join("", str);
        
        return answer;
    }
}
